eppoclient/rules.go

```diff
@@ -159,38 +159,6 @@ func isOne(attributeValue interface{}, s string) bool {
 	}
 }
 
-func promoteInt(i interface{}) int64 {
```

Comment:
Moved `promoteInt()` and `promoteUint()` to the bottom of the file (along with a newly created `promoteFloat()`) as they are now used by `toFloat64()` and Clean Code suggests having shared helper methods after all places they are used